### PR TITLE
Add Cell Ranger mkfastq

### PR DESCRIPTION
**Pull request by lebovic**
Adds:
- Cell Ranger mkfastq

### CODE REVIEW

**Comment by lebovic** [comment]
@bcai2 requesting your preliminary review; this is still untested

**Review by bcai2 — APPROVED** [review]
The naming of `_validate_and_prepare()` is confusing, esp. next to `_validate_and_prepare_inputs()`. Otherwise, LGTM

**Comment by lebovic** [comment]
> The naming of `_validate_and_prepare()` is confusing, esp. next to `_validate_and_prepare_inputs()`. Otherwise, LGTM

Fixed by renaming `_validate_and_prepare()` back to `_validate_args()`, renamed `_validate_and_prepare_inputs()` to `_prepare_inputs()`, and moved the call of `_prepare_inputs()` out of `_validate_args()` to `run()`.

File: toolchest_client/files/general.py [hunk 1]
```diff
@@ -69,6 +74,26 @@ def files_in_path(files):
     return more_files
 
 
+def compress_files_in_path(file_path):
+    """Returns a tarred and compressed file containing the contents of a directory.
+
+    WARNING: this is NOT thread-safe, as shutil.make_archive is not thread safe.
```
[the remark this answers is not in the document]
Comment (lebovic) [reply toolchest_client/files/general.py:80]: This means we can't run many compressions in parallel (for now)

**Review by bcai2 — APPROVED** [review]
LGTM, just bump the package version # and you should be good to merge.

**Pull request merged**